plugins/BEdita/Core/tests/TestCase/Model/Action/AddRelatedObjectsActionTest.php

```diff
@@ -110,6 +110,26 @@ public function invocationProvider()
                     ],
                 ],
             ],
+            'update, more than one related entities' => [
+                [2, 3],
+                'Profiles',
```

Comment:
:thinking: I think the test passes without the modify too.

`prepareData()` immediately return if

```php
if (empty($data['entity']) || !($data['entity'] instanceof Folder) || $this->Association->getName() !== 'Parents') {
    $this->setupPriority($data);
    return $data;
}
```

So I think the test should be done with `Folders` as main object and `parents` as relation.
But in this way the save on folders should be fail due to validation rule https://github.com/bedita/bedita/blob/4-cactus/plugins/BEdita/Core/src/Model/Table/FoldersTable.php#L82

If the save works we have a problem. In this case we should throw an Exception if `$relatedEntities` is an array with more than one value.

Reply:
changing this as suggested, an exception is thrown (as you said, @batopa ).
```
BEdita\Core\Test\TestCase\Model\Action\AddRelatedObjectsActionTest::testInvocation with data set "update, more than one related entities" (array(2, 3), 'Folders', 'parents', 4, array(array(2, 1, null), array(1, 1, array('value'))))
Cake\Database\Exception: Cannot describe trees. It has 0 columns.
```

Reply:
I changed this part, to throw an exception on "more than one parent provided", for folders.